pkg/ccl/backupccl/backuputils/testutils.go

```diff
@@ -172,3 +172,30 @@ func VerifyBackupRestoreStatementResult(
 
 	return nil
 }
+
```

Comment:
@adityamaru i feel uneasy about merely logging these errors. One alternative is to pass a `dontCheckInvalidDescriptors` bool to each backupTestClusterSetup call, which prevents a CheckForInvalidDescriptors call. Is that preferred?

Reply:
Why don't we `t.Fatal` if we see an invalid descriptor so the test fails?

Reply:
Done.